lib/open-amp/vendor_rsc_table/nxp_resource_table.c

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
could be nice to use similar tree than dts folder  by naming the folder `vendor` instead of `vendor_rsc_table`